ds701_book/jupyter_notebooks/19-Regression-III-More-Linear.ipynb

```diff
@@ -7,39 +7,27 @@
         "---\n",
```

Comment:
<p>Can we add the 2nd plot that shows the solution like in the previous example? Or would it not fit?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/DS701-Course-Notes/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Added.</p>